theory/energetic sources/supernovae.md

```diff
@@ -7,3 +7,22 @@ nav_order: 4
 ---
 
 ## Supernovae
+
+A supernova is a powerful explosion that marks the final stage of a star's life. As the star approaches its death, its core begins to collapse under immense pressure. This collapse forces protons and electrons to combine, forming neutrons. The star's outer layers are then expelled into space in a spectacular explosion.
+
+Supernovae release huge amounts of energy. The amount of energy released can go up to 10^51 ergs.
+
+{: .fun}
+>A single supernova can release more energy in a few weeks than the Sun will emit over its entire 10-billion-year lifetime.
+
+Supernovae are the sole reason for heavy elements in our universe. It provides the right conditions for the nucleosynthesis of these elements.
+
+### Types of Supernova
+
+
+1. Type II Supernova:
+When a massive star(Mass more than 8 solar masses) collapses under its own gravity to explode. This explosion is the result of a sudden shockwave formed by the rebound of the collapsing outer layers of the star after they collide with the newly formed dense core.
+
+2. Type I Supernova:
+When a dwarf star in its [binary system](../special%20stars/binary%20stars.html) accretes material from its companion star it can increase its mass beyond the Chandrasekhar limit.After exceeding the limit the star becomes unstable and it triggers a thermonuclear explosion resulting in a supernova.
```

Comment:
Shouldn't Type I supernova be mentioned before Type II?

Reply:
changed the order